src/layout/Accordion.js

```diff
@@ -0,0 +1,120 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["../common/HTMLWidget", "../form/Form", "../form/Input", "../common/FAChar", "css!./Accordion"], factory);
+    } else {
+        root.layout_Accordion = factory(root.common_HTMLWidget, root.form_Form, root.form_Input, root.common_FAChar);
+    }
+}(this, function (HTMLWidget, Form, Input, FAChar) {
+    function Accordion() {
+        HTMLWidget.call(this);
+
+        this._tag = "div";
+        this._isClosed = false;
+    }
+    Accordion.prototype = Object.create(HTMLWidget.prototype);
+    Accordion.prototype.constructor = Accordion;
+    Accordion.prototype._class += " layout_Accordion";
+
+    Accordion.prototype.publish("content", [], "widgetArray", "Array of widgets",null,{tags:["Basic"]});
+    Accordion.prototype.publish("title", "", "string", "Title of collapsible section",null,{tags:["Private"]});
+    Accordion.prototype.publish("openIcon", "\uf147", "string", "Icon to display when list is open",null,{tags:["Private"]});
+    Accordion.prototype.publish("closedIcon", "\uf196", "string", "Icon to display when list is closed",null,{tags:["Private"]});
+    Accordion.prototype.publish("titleFontColor", "#FFFFFF", "html-color", "Title font color",null,{tags:["Private"]});
+    Accordion.prototype.publish("titleBackgroundColor", "#333333", "html-color", "Title background color",null,{tags:["Private"]});
+    
+    Accordion.prototype.publish("defaultCollapsed", false, "boolean", "Collapsed by default if true",null,{tags:["Private"]});
+    
+    Accordion.prototype.pushListItem = function (widget,prepend,protect) {
+        var contentArr = this.content();
+        
+        widget._protected = protect;
+        
+        if(prepend){
+            contentArr = [widget].concat(contentArr);
+        } else {
+            contentArr.push(widget);
+        }
+        this.content(contentArr);
+        return this;
+    };
+    
+    Accordion.prototype.clearListItems = function () {
+        var arr = [];
+        for(var i in this.content()){
+            if(this.content()[i]._protected){
+                arr.push(this.content()[i]);
+            }
+        }
+        this.content(arr);
```

Comment:
"_protected" items are immune to the **clearListItems()** method. It's being used to prevent the PropertyEditor's "Editor Settings" section from being cleared each time the PropertyEditor updates.